services/environment/__init__.py

```diff
@@ -303,4 +303,4 @@ def weather(self, site: Site,
     app = FastAPI()
     app.add_route('/graphql', graphql_app)
     app.add_websocket_route('/graphql', graphql_app)
-    uvicorn.run(app, host='127.0.0.1', port=8000)
+    uvicorn.run(app, host='127.0.0.1', port=8001)
```

Comment:
You can remove this after https://github.com/gemini-hlsw/scheduler/pull/165 is approved. Should we have OCS Env still running on 8001 to avoid conflicts with the Scheduler GraphQL server or will it not matter? I'm not sure about the deployment yet.